packages/ember-runtime/lib/system/core_object.js

```diff
@@ -267,6 +254,10 @@ CoreObject.PrototypeMixin = Mixin.create({
     @method init
   */
   init: function() {},
+  __defineNonEnumerable: function(property) {
+    o_defineProperty(this, property.name, property.descriptor);
+    //this[property.name] = property.descriptor.value;
```

Comment:
What is really going on here? Why not use the commented line instead of defineProp? Mixins don't care if they have extra enumerable props, yes? This line is responsible for a very large number of defineProp calls in my testing.